protos/platform/v0/platform.proto

```diff
@@ -69,3 +75,29 @@ message GetIdentityIdByFirstPublicKeyRequest {
 message GetIdentityIdByFirstPublicKeyResponse {
   string id = 1;
 }
+
+message PublicKeyHashIdentityPair {
+  bytes public_key_hash = 1;
```

Comment:
Does it make sense, since Identity contains keys so you always know which identity obtain by which key. Just extra data?